trax/models/research/bert.py

```diff
@@ -255,6 +269,13 @@ def init_weights_and_state(self, input_signature):
     if self.init_checkpoint is None:
       return
 
+    if self.init_checkpoint.endswith('.pkl.gz'):
+      # file is a model saved in trax
+      model2 = BERT(head=functools.partial(BERTPretrainingHead, n_classes=2))
+      model2.init_from_file(self.init_checkpoint)
```

Comment:
Initializing models should be done using the default interface.

Reply:
I'm not sure if this is possible with default interface, the problem is that loaded model has bert core and pretraining head and we're interested only in loading the core, not the head. If it is possible could you give me a hint or direct me to some snippet that does something similar?